### ISSUE TITLE
Stable_baselines3

### ISSUE THREAD

**Issue by Omikaa** [btx0424/OmniDrones#69]
Hello,can I use stable_baselines3 to work with the omnidrones platform. My algo work was on the stable_baselines3,and I want to migrate it. How can I modify the algo part in Omnidrones?

**Comment by btx0424** [comment]
I don't think you should use SB3 here. Not because it is not good, but because it is not designed for a large number of parallel environments. That's also why I did not follow the old OpenAI Gym interface. I would suggest implementing your algorithm following the `torchrl` interface.

**Comment by Omikaa** [comment]
Thanks a lot, I get your suggestion.Maybe after I finish my latest work, I will consider using omnidrones.
